Mathlib/Order/Interval/Set/Basic.lean

```diff
@@ -38,35 +38,35 @@ section Preorder
 
 variable [Preorder α] {a a₁ a₂ b b₁ b₂ c x : α}
 
-/-- Left-open right-open interval -/
+/-- Left-open right-open interval $(a, b)$ -/
```

Comment:
Maybe
```suggestion
/-- `Icc a b` is the left-open right-open interval $(a, b)$ -/
```
since otherwise you don't actually see what `a` and `b` refer to. Feel free to wait for other reviewer votes before changing it.

Reply:
You have a typo in your suggestion: should be `Ioo`, not `Icc`.